config/connect-mirror-maker.properties

```diff
@@ -18,18 +18,40 @@
 # Run with ./bin/connect-mirror-maker.sh connect-mirror-maker.properties 
 
 # specify any number of cluster aliases
-clusters = A, B, C
+clusters = A, B 
 
 # connection information for each cluster
-A.bootstrap.servers = A_host1:9092, A_host2:9092, A_host3:9092
-B.bootstrap.servers = B_host1:9092, B_host2:9092, B_host3:9092
-C.bootstrap.servers = C_host1:9092, C_host2:9092, C_host3:9092
+# This is a comma separated host:port pairs for each cluster
+# for e.g. "A_host:9092, A_host2:9092, A_host3:9092"
+A.bootstrap.servers = localhost:9092
+B.bootstrap.servers = localhost:10092
 
 # enable and configure individual replication flows
 A->B.enabled = true
-A->B.topics = foo-.*
-B->C.enabled = true
-B->C.topics = bar-.*
+
+# regex which defines which topics gets replicated. For eg "foo-.*"
+A->B.topics = .*
+
+B->A.enabled = true
```

Comment:
Looks like below two lines also enable replication from B to A (in addition to existing replication from A to B).
Maybe to make this configuration more straightforward it's enough to only replicate from A to B? (this configuration previously had a similar setup)
What do you think?

Reply:
It's a lot more common to have bidirectional replication. I think having A->B and B->A is more illustrative of a real configuration, without being much more complicated.